server/webhook.go

```diff
@@ -74,7 +123,7 @@ func (p *Plugin) handleMeetingEnded(w http.ResponseWriter, r *http.Request, body
 		return
 	}
 
-	meetingPostID := webhook.Payload.Object.ID
+	meetingPostID := webhook.Payload.Object.UUID
```

Comment:
It seems this will have some backwards compatibility issues, though I don't think that's a big issue in this plugin. The posts or more-or-less "ephemeral" in usage, as far as looking them up in the kvstore after a meeting ends

Reply:
Yes, exactly, this only can affect meetings that were running during the plugin upgrade, this ids are stored in the KV store in a transient mode. It is no longer that way, we are storing them and leaving them there. And they are "invalidated" automatically because they have an "Expire" date in the KV store of 24 hours.

This also fixes the problem of having a meeting open, close it, and open it again quickly, that would end up referring to the same meeting ID, while the UUID is per session. This is more important now because the information to track the meeting needs to be alive more time to put the recording and the transcription there.